independent-publisher-connectors/Proposal - RegEx Matching (Independent Publisher)/readme.md

```diff
@@ -0,0 +1,25 @@
+## RegEx Matching
+This connector extends Power Automate capabilities for performing tests whether a text string matches a pattern. There are pre-built patterns and also an option for user to provide their own regular expressions (regex) for matching the text string. 
+
+## Publisher
+Mitanshu Garg
+
+## Prerequisites
+There are no prerequisities for this connector
+
+## Supported Operations
+| Action | Description |
+| --- | --- |
+| `valid Email` | Returns `true` if the text string is a valid email address |
+| `valid GUID` | Returns `true` if the text string is in a valid GUID format |
+| `valid SSN` | Returns `true` if the text string is in United States Social Security Number format |
+| `valid Credit Card` | Returns `true` if the text string is in credit card number format |
+| `valid DATETIME` | Returns `true` if the text string is in YYYYMMDDHHMMSS format |
+| `contains Digit` | Returns `true` if the text string contains a digit (0-9) anywhere in the input |
+| `startsWith` | Returns `true` if the text string starts with the given keyword |
+| `endsWith` | Returns `true` if the text string ends with the given keyword |
+| `match pattern` | Returns `true` if the text string matches the custom regex pattern |
+
+## Limitations and Exclusions
```

Comment:
@happyume please format the readme.md file according to [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md). All the required headings should be present, they also should be placed in the correct order, like in the template.

Reply:
This is now updated